jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -137,10 +152,18 @@ public void addLayerDiffId(DescriptorDigest diffId) {
     rootfs.diff_ids.add(diffId);
   }
 
+  public void addHistory(HistoryEntry historyEntry) {
```

Comment:
nit: since `history` is a plural list of `HistoryEntry`s, perhaps this method name should be `addToHistory` instead